spec/use_case/count_views_spec.rb

```diff
@@ -0,0 +1,229 @@
+describe UseCase::CountViews do
+  let(:file_gateway) { double('Gateway::File') }
+  before do
+    allow(file_gateway).to receive(:readlines)
+      .and_return(lines_array)
+  end
```

Comment:
Usually, use case tests are using a fake gateway like this for performance reasons.